### PR TITLE
Matching burst header to expected format

### PR DESCRIPTION
**Pull request by sivamu**
Closes: https://github.com/retaildevcrews/wcnp/issues/175

### CODE REVIEW

File: src/Controllers/HealthzController.cs
Comment (sivamu) [inline src/Controllers/HealthzController.cs:62]: The header is output everytime. Should I add a config to enable/disable the header?

Comment (jofultz) [reply src/Controllers/HealthzController.cs:62]: Open an issue in triage for it.  I think we should.

File: src/Core/CommandLine.cs
Comment (jofultz) [inline src/Core/CommandLine.cs:101]: I feel like we should be able to reuse the app name from here:

```csharp

RootCommand root = new RootCommand            
{                
   Name = "Ngsa.Application", 
   Description = "NGSA Validation App",
   TreatUnmatchedTokensAsErrors = true,
};
```

Let's use that app name for the moment if it presents no issues and open another issue to discuss whether we should pass in the burst service name, derive it programmatically, or do something like pass in a suffix.

If it does present an issue, go with what you have, but open another issue as Design Review to look at this together.

**Review by jofultz — APPROVED** [review]

File: src/Controllers/HealthzController.cs
Comment (bartr) [reply src/Controllers/HealthzController.cs:62]: it's in the healthz controller, so it's only output on /healthz/* calls.  I think that's what we want.

File: src/Core/CommandLine.cs
Comment (bartr) [reply src/Core/CommandLine.cs:101]: you should be able to extend Version to pull the info when we do the reflection for the version info

Comment (bartr) [reply src/Core/CommandLine.cs:101]: agree that I wouldn't pass as param

**Review by bartr — APPROVED** [review]
approved with comments

**Comment by sivamu** [comment]
Adding blocked label for now, need to discuss items in issue #178 for design review before merging.

**Comment by bartr** [comment]
We don't have to block the PR - we can always update the code after design review.  It's also valid to wait.

**Comment by sivamu** [comment]
Remove block but will wait until after Design Review

**Pull request merged**
